.github/manual-toolkit-push.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+
+TYPE=("cuda" "ml")
+TARGETS=()
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+  --cuda-toolkit)
+    TYPE=("cuda")
+    shift 1
+    ;;
+  --ml-toolkit)
+    TYPE=("ml")
+    shift 1
+    ;;
+  --target)
+    if [[ "$TARGET" != "cpu" && "$TARGET" != "gpu" && "$TARGET" != "jetson" ]]; then
+      echo "Invalid target specified. Use 'cpu', 'gpu', or 'jetson'."
+      exit 1
+    fi
+    TARGETS=("$TARGET")
+    shift 2
+    ;;
+  esac
+done
+
+echo $SCRIPT_DIR
+
+for type in "${TYPE[@]}"; do
+  echo "Building $type toolkit..."
+
+  if [ "$type" = "cuda" ]; then
+    if [ ${#TARGETS[@]} -eq 0 ]; then
+      TARGETS=("default" "jetson")
+    fi
+
+    for target in "${TARGETS[@]}"; do
+      echo "Building for target: $target"
+      if [ "$target" = "jetson" ]; then
+        echo "Building for Jetson..."
+        bash $SCRIPT_DIR/../toolkits/build.sh --cuda-toolkit  \
+          --target jetson \
+          --ubuntu-version 22.04 \
+          --python-version 3.10 \
+          --tensorrt-image nvcr.io/nvidia/l4t-tensorrt \
+          --tensorrt-image-tag r8.6.2-devel
+      else
+        echo "Building for generic platform..."
+        bash $SCRIPT_DIR/../toolkits/build.sh --cuda-toolkit
+      fi
+    done
+  elif [ "$type" = "ml" ]; then
+    if [ ${#TARGETS[@]} -eq 0 ]; then
+      TARGETS=("cpu" "gpu" "jetson")
+    fi
+
+    for target in "${TARGETS[@]}"; do
+      if [ "$target" = "jetson" ]; then
+        echo "Building for Jetson..."
+        bash $SCRIPT_DIR/../toolkits/build.sh --ml-toolkit \
+          --target jetson \
+          --ubuntu-version 22.04 \
+          --python-version 3.10 \
+          --tensorrt-image nvcr.io/nvidia/l4t-tensorrt \
+          --tensorrt-image-tag r8.6.2-devel
+      elif [ "$target" = "cpu" ]; then
+        echo "Building for CPU..."
+        $SCRIPT_DIR/../toolkits/build.sh --ml-toolkit --target cpu
+      elif [ "$target" = "gpu" ]; then
+        echo "Building for GPU..."
+        $SCRIPT_DIR/../toolkits/build.sh --ml-toolkit --target gpu
+      fi
+    done
+  fi
+
+  # push to registry
+  docker image push --all-tags ghcr.io/aica-technology/toolkits/$type
```

Comment:
this requires a clean docker image list though. Perhaps it's for as long as the process is manual in any case